client/sections.js

```diff
@@ -5,6 +5,13 @@ const fs = require( 'fs' ); // eslint-disable-line import/no-nodejs-modules
 const path = require( 'path' ); // eslint-disable-line import/no-nodejs-modules
 
 const sections = [
+	{
+		name: 'calypso-entry',
```

Comment:
I think the word `calypso` here is redundant. Should we also make it more obvious that this is the root? Should we name this `root` instead of `calypso-entry`?

Reply:
also update to root-entry